ember_debug/models/profile-manager.js

```diff
@@ -5,6 +5,74 @@ const {
   run: { later, scheduleOnce },
 } = Ember;
 
+function getUnfilteredRoots(first, last, closest) {
+  const roots = [];
+  let start = null;
+  let end = null;
+  for (let i = 0; i < closest.length; i++) {
+    if (closest.item(i) === first.node) start = i;
+    else if (closest.item(i) === last.node) end = i;
+  }
+
+  if (start === null || end === null) return [];
+
+  for (let i = start; i <= end; i++) roots.push(closest.item(i));
+
+  return roots;
+}
+
+function _findRoots({ first, last, parent }) {
+  const closest = parent.childNodes;
+  if (first.node === last.node) return [first.node];
```

Comment:
looks like here we also should check for `nodeType === 1` and return empty array if not matched

Reply:
updated.